_docs/1-demo/bookinfo.md

```diff
@@ -182,7 +184,8 @@ from your browser and you should see the bookinfo application `productpage`
 displayed.  Notice that the `productpage` is displayed, with no rating
 stars since `reviews:v1` does not access the ratings service.
 
-**Note**: Replace `GATEWAY_HOST_PORT` with the output of the following command:
+**Note**: Replace `GATEWAY_URL` with the output of the following command:
```

Comment:
Ports are an optional part of a URL:

https://en.wikipedia.org/wiki/Uniform_Resource_Locator#Syntax

I think it would be less confusing to rename the environment variable to `GATEWAY_URL` a cross the board.